### ISSUE TITLE
Questions about the paper

### ISSUE THREAD

**Issue by Gyuyeong** [ShannonAI/GPT-CLS-CARP#1]
After reading the paper, I have some questions:

1. Is CARP applied on an already fine-tuned LLM like Chat GPT? If so, if I am trying to apply this concept in a model that has not been fine-tuned at all (for example GPT variants that can be found in Huggingface), how should I prepare the training data to fine-tune the LLM so that CARP can be effectively applied?
2. I do not understand what the paper says regarding using training sets. From what I understand, there is a training set, and you sample some of them out using SimCSE for few-shot learning demonstration examples. However, I do not understand where the training set is used other than when sampling out the few-shot examples. Were they used to fine-tune the LLM and you keep them for future use?

I apologize if I asked anything that was already mentioned in the paper and I was not paying close attention to it. Thank you in advance

**Comment by zhaozhaoxia** [comment]
请讲中文

**Comment by PeterXiaTian** [comment]
看完这篇文章，个人觉得作者没有用训练集微调任何大模型，只是挑选了部分样本来做few-shot.　但是这里挑选样本的时候比较疑惑，比如用simces来选择和问题相似的文本，问题样本来自哪里？？［mask了部分带label的样本吗？]  但是如果没有任何先验样本的话，这个该如何处理了？

**Comment by pranerd** [comment]
table 4 in this paper show that as the training set grows，16，128,256,512,1024, the carp models accuracy increase, how did it use the increased training set, did LLM model fine-tune on the training set? 
![screenshot_36](https://github.com/ShannonAI/GPT-CLS-CARP/assets/19358928/ac73eaf7-0be4-4a6c-bdf1-bbb9001bb51e)
